src/common/debugSession.ts

```diff
@@ -42,7 +43,11 @@ export interface ILaunchRequestArguments
   /** An absolute path to the simulate response to debug. */
   simulateTraceFile: string;
   /** An absolute path to the file which maps programs to source maps. */
-  programSourcesDescriptionFile: string;
+  programSourcesDescriptionFile?: string;
+  /** JSON encoded content of the program sources description file. */
+  programSourcesDescription?: string;
```

Comment:
This should be an object of type `ProgramSourceEntryFile` rather than the JSON encoded equivalent